analyses/compare-gistic/results/lgat_gene_level_focal_cn_counts.tsv

```diff
@@ -0,0 +1,119 @@
+gene_symbol	detection_peak	loss	gain	amplification
+ACP6	Deletion Peak 1	243	243	247
```

Comment:
I think we have 242 LGAT samples, so this is missing some kind of `distinct` step I bet.